frontend/public/components/volumes-table.tsx

```diff
@@ -52,39 +55,45 @@ const ContainerLink: React.FC<ContainerLinkProps> = ({name, pod}) => <span class
 </span>;
 ContainerLink.displayName = 'ContainerLink';
 
-const VolumeRow: React.FC<VolumeRowProps> = ({value, pod}) => {
-  const kind = _.get(getVolumeType(value.volume), 'id', '');
-  const loc = getVolumeLocation(value.volume);
-  const name = value.name;
-  const permission = value.readOnly ? 'Read-only' : 'Read/Write';
+const VolumeRow: React.FC<VolumeRowProps> = ({volume, resource}) => {
+  const type = _.get(getVolumeType(volume.volume), 'id', '');
```

Comment:
`volume.volume` is really awkward. I'm not sure this is better than `value.volume`

Reply:
Let's try volumeData.